### PR TITLE
Ceara/aitt 467 update dropdown student status

### PR DESCRIPTION
**Pull request by cearachew**
Updates the student selection drop down to include the students' submit statuses. Replaces the default select with React Select to allow for styling options. 

Without any students selected:
Screenshot 2024-03-01 at 11 51 00 AM

Student Dropdown:
Screenshot 2024-03-01 at 12 19 42 PM

With a student selected:
Screenshot 2024-03-01 at 12 19 59 PM

Section dropdown:
Screenshot 2024-03-01 at 12 20 09 PM

## Links

Jira ticket: https://codedotorg.atlassian.net/browse/AITT-467
Figma: https://www.figma.com/file/iQgdugbutqjoPuf7GMUxR7/AI-Rubrics-Prototype?type=design&node-id=3714%3A3436&mode=design&t=6yf4wXIxQMynSoxc-1

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by cearachew** [comment]
@Nokondi can you take a look at this?

File: apps/src/templates/rubrics/rubrics.module.scss [hunk 1]
```diff
@@ -912,3 +916,61 @@ i.infoAlertIcon {
...
+  background-color: $light_gray_100;
+  border-radius: 20px;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 4px;
+  display:inline;
+  p {
+    margin: 0,0,0,0;
+  }
+}
```
Comment (Nokondi) [inline apps/src/templates/rubrics/rubrics.module.scss:965]: Check the bubble styling in the figma. The text looks a little off. You may need to modify the padding-top to bump it down to the center.

Comment (Nokondi) [reply apps/src/templates/rubrics/rubrics.module.scss:965]: The height may also need to be set to a constant value.

Comment (cearachew) [reply apps/src/templates/rubrics/rubrics.module.scss:965]: Updated the styling
Screenshot 2024-03-06 at 1 52 52 PM

Comment (cearachew) [reply apps/src/templates/rubrics/rubrics.module.scss:965]: lol I forgot to push, but it's there now
Screenshot 2024-03-06 at 2 58 42 PM

**Review by Nokondi — APPROVED** [review]
Great work on this!

**Pull request merged**